env.env

```diff
@@ -0,0 +1,7 @@
+DOMAIN=localhost

```

Comment:
`.env` data should not be pushed, this should be kept on your system. There is already a rule in the `.gitignore` file that should prevent this from happening. But in your case, it looks like it did not work because the name of the file does not match was is expected. Environment variables should be called `.env` not `env.env`. I'm assuming this was left over before you added the correct file. If that's the case, this file should be deleted.

Reply:
Yeah, this file was leftover and was meant to be deleted. Fixed!